packages/components/psammead-podcast-promo/src/components/card-image-wrapper.jsx

```diff
@@ -4,11 +4,15 @@ import {
   GEL_GROUP_2_SCREEN_WIDTH_MIN,
   GEL_GROUP_3_SCREEN_WIDTH_MIN,
   GEL_GROUP_4_SCREEN_WIDTH_MIN,
+  GEL_GROUP_B_MIN_WIDTH,
 } from '@bbc/gel-foundations/breakpoints';
 
 const CardImageWrapper = styled.div`
   flex: 0 0 5.5rem;
```

Comment:
Think you can delete this line now as the style will no longer be applied because of the `display: none` 👍